packages/react/CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing
+
+## Getting Started
+
+1. Navigate to the _root_ of your local clone of [aws-amplify/amplify-ui](https://github.com/aws-amplify/amplify-ui)
+1. Run `yarn setup`
+1. Run `yarn react dev`
+
+This will start building `@aws-amplify/ui-react` in watch mode. To test your changes, you can utilize [`examples/next`](../../examples/next) to run examples on `next.js`. Please see examples [README](../../examples/README.md) and e2e [README](../e2e/README.md#contributing) to get started.
```

Comment:
Later, we should consider adding alternative ways to test your changes using `yarn link` or `verdaccio`. `examples` and `e2e` need much setup, and contributors might just want to test their sample app directly.